### DISCUSSION TITLE
Checkbox list scrollable

### DISCUSSION THREAD

**Discussion by sasori13L** [filamentphp/filament#11571] [Help] [answered]
### Package

Form builder

### Package Version

v3.2.0

### How can we help you?

How to add scrollbar within checkbox list not entire slideover modal?
![image](https://github.com/filamentphp/filament/assets/16425810/26a9973d-c268-4444-9b59-e3515a9b3189)

When I scroll down it covers the other form contents.
![image](https://github.com/filamentphp/filament/assets/16425810/7714f7e9-b105-4d66-a27d-8b554ea37365)

**Comment by zeeshantariq08** [comment, marked as answer]
you can use a checkbox list
```
CheckboxList::make('lists')
                                        ->label('Associated Requirements')
                                        ->relationship('requirements')
                                        ->options(function (callable $get) {
                                         return List::pluck('name','id')

                                        })
                                        ->columns(2)
                                        ->gridDirection('row')
                                        ->searchable()
                                        ->dehydrated()
                                        ->searchDebounce(500)
                                        ->searchPrompt('Search list')
                                        ->bulkToggleable()
                                        ->selectAllAction(
                                            fn(Action $action) => $action->label('Select all List'),
                                        )
                                        ->extraAttributes([
                                            'style' => 'max-height: 300px !important; overflow-y: scroll !important;',
                                        ]),
```

**Reply by sasori13L** [reply]
Thank you for this answer. 
Tried it but some methods does not actually work on my part. ended up just using the 
`->gridDirection('row')
 ->extraAttributes([ 'style' => 'max-height: 300px !important; overflow-y: scroll !important;',])`.

Thank you so much indeed! ^_^

**Comment by leandrocfe** [comment]
Why not [splitting options into columns](https://filamentphp.com/docs/3.x/forms/fields/checkbox-list#splitting-options-into-columns)?
